centrifuge/centrifuge-chain
# Release v0.13.0

# Description

* Removes migrations which are expected to have been executed
* Bumps spec version to 1300
* Updates deps to enable metadata hash check as recommended [on Element](https://matrix.to/#/!pYjTIdxhocIQxcOXdb:matrix.parity.io/$rgj5rvCJXXitBp49-8Umx6bD-VMqoanZFefY7vgiSaw?via=matrix.parity.io&via=matrix.org&via=parity.io)
* Updates srtool to latest version
* Adds `on-chain-release-build` to srtool build opts
  * Longterm we should switch to using the explicit github action: https://github.com/polkadot-fellows/runtimes/blob/bb52c327360d1098d3b3d36f4eafb40a74636e80/.github/workflows/release.yml#L59

*Pipeline enhancements*
- Limit sanity_checks to 90m to avoid consuming minutes on stuck jobs
- Modify wasm build to output srtool information
- Upload artifacts when creating a release (wasm and json summary)
- Upload json summary to GCP bucket too
- Add a manual workflow to delete extra containers from GH registry (verified working)
- Remove updating the release with the docker tag as appended comment (preferred to do manually)

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
